help/en/html/doc/Technical/Eclipse.shtml

```diff
@@ -152,14 +152,24 @@
 
 				<li>Click "Run" to run the application. Eclipse will retain the
 					build in the Run menu.</li>
-			</ol>
-
-			If during the launch of DecoderPro or PanelPro the program hangs, or
+               </ol>
```

Comment:
Not a big problem, but the previous HTML code have indentation by tabs while your new code have indentation by spaces. If it's easy to fix, it might be good to change the tabs to spaces.

When I look att PR:s on Github that seems to have the wrong indentation, I use the mouse to select the spaces before the text on a single line. By doing that, tabs gets "visible" since one tab seems to be 8 spaces on Github.